packages/ai-core/src/browser/ai-configuration/agent-configuration-widget.tsx

```diff
@@ -100,6 +101,9 @@ export class AIAgentConfigurationWidget extends ReactWidget {
                         </li>
                     )}
                 </ul>
+                <div className='configuration-agents-add'>
+                    <button style={{ marginLeft: 0 }} className='theia-button main' onClick={() => this.addCustomAgent()}>Add Custom Agent</button>
+                </div>
```

Comment:
If there are already custom agents then this should be called  "Edit Custom Agents" as it does no longer add one